.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Linting and unit tests
+
+# only run this workflow on new commits to main
+# or PRs into main
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+  schedule:
+    # Run every Monday morning at 11:00a UTC, 6:00a CST
+    - cron: '0 11 * * 1'
+
+jobs:
+  test:
+    name: ${{ matrix.task }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - task: linting
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Set up Python 3.7
+        if: matrix.task != 'todo-checks'
+        uses: s-weigand/setup-conda@v1
+        with:
+          python-version: 3.7
+      - name: linting
+        if: matrix.task == 'linting'
+        shell: bash
+        run: |
+          pip install --upgrade black flake8 pylint
+          make lint
```

Comment:
for this PR could a minimal test that the package is installable be added? it would be lightweight to add but would give us some coverage and at least catch big problems like errors in setup.py

https://github.com/saturncloud/prefect-saturn/blob/main/.github/workflows/main.yml#L48-L53

Reply:
Yeah good idea. I think we can also test this up in integration tests.